backend/src/main/java/com/benine/backend/database/MySQLDatabase.java

```diff
@@ -0,0 +1,247 @@
+          + preset.getPan() + "," + preset.getTilt() + "," + preset.getZoom()
+          + "," + preset.getFocus() + "," + preset.getIris() + "," + auto + ")";
+      statement.executeUpdate(sql);
+      sql = "INSERT INTO presetsdatabase.camerapresets VALUES(" + cameraPresetNumber + "," + camera
+          + "," + presetId + ")";
+      statement.executeUpdate(sql);
+      statement.close();
+      presetId++;
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+
+  }
+
+  @Override
+  public void deletePreset(int camera, int cameraPresetNumber) {
+    try {
+      Statement statement = connection.createStatement();
+      String sql = "DELETE FROM presetsdatabase.camerapresets WHERE Camera_ID = " + camera
+          + " AND CameraPresetID = " + cameraPresetNumber;
+      statement.executeUpdate(sql);
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+  }
+
+  @Override
+  public void updatePreset(int camera, int cameraPresetNumber, DatabasePreset preset) {
+    int auto = 0;
+    if (preset.isAutofocus()) {
+      auto = 1;
+    }
+    try {
+      Statement statement = connection.createStatement();
+      String sql = "INSERT INTO presetsdatabase.presets VALUES(" + presetId + "," + preset.getPan()
+          + "," + preset.getTilt() + "," + preset.getZoom() + "," + preset.getFocus()
+          + "," + preset.getIris() + "," + auto + ")";
+      statement.executeUpdate(sql);
+      sql = "UPDATE presetsdatabase.camerapresets SET Presets_ID = " + presetId
+          + "WHERE Camera_ID = " + camera + "AND CameraPresetID = " + cameraPresetNumber;
+      statement.executeUpdate(sql);
+      presetId++;
+      statement.close();
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+  }
+
+  @Override
+  public DatabasePreset getPreset(int camera, int cameraPresetNumber) {
+    DatabasePreset preset = new DatabasePreset(0, 0, 0, 0, 0, false);
+    try {
+      Statement statement = connection.createStatement();
+      String sql = "SELECT pan, tilt, zoom, focus, iris, autofocus FROM presetsDatabase.presets "
+          + "JOIN presetsDatabase.camerapresets ON presetsDatabase.camerapresets.Presets_ID = "
+          + "presetsDatabase.presets.ID " + "WHERE presetsDatabase.camerapresets.Camera_ID = "
+          + camera + " AND presetsDatabase.camerapresets.CameraPresetID = " + cameraPresetNumber;
+      ResultSet resultset = statement.executeQuery(sql);
+      if (resultset.next()) {
+        preset.setPan(resultset.getInt("pan"));
+        preset.setTilt(resultset.getInt("tilt"));
+        preset.setZoom(resultset.getInt("zoom"));
+        preset.setFocus(resultset.getInt("focus"));
+        preset.setIris(resultset.getInt("iris"));
+        preset.setAutofocus(resultset.getInt("autofocus") == 1);
+      }
+      resultset.close();
+      statement.close();
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+    return preset;
+  }
+
+  @Override
```

Comment:
This method contains duplicate code from the previous method, can this be resolved?